__tests__/app.test.js

```diff
@@ -115,4 +115,52 @@ describe('/api/reviews/:review_id', () => {
                 });
         });
     });
+});
+
+describe('/api/reviews/:review_id/comments', () => {
+    describe("GET", () => {
+        test("should return the review from the table if the Id is valid", () => {
+            return request(app)
+                .get("/api/reviews/2/comments")
+                .expect(200)
+                .then(({ body }) => {
+
+                    //expect(body.review.length).toBeGreaterThan(0)
```

Comment:
I would keep this in and assert for shape like you've done previously using `expect.any` for each property - apart from the review_id - that should definitely be 2 👍